docs/source/changelog.rst

```diff
@@ -8,56 +8,148 @@ Changes in nbconvert
+
+The ``raises-exception`` cell tag (``nb.cells[].metadata.tags[raises-exception]``) allows for cell exceptions to not halt execution. The tag is respected in the same way by nbval and other notebook interfaces. ``nb.metadata.allow_errors`` will apply this rule for all cells. This feature is toggleable with the ``force_raise_errors`` configuration option.
+Errors from executing the notebook can be allowed with a ``raises-exception`` tag on a single cell, or the ``allow_errors`` configurable option for all cells. An allowed error will be recorded in notebook output, and execution will continue.
+If an error occurs when it is not explicitly allowed, a ``CellExecutionError`` will be raised.
+If ``force_raise_errors`` is True, ``CellExecutionError`` will be raised for any error that occurs while executing the notebook. This overrides both the ``allow_errors`` option and the `raises-exception` cell tags.
+
+See :ghpull:`867`, :ghpull:`703`, :ghpull:`685`, :ghpull:`672`, and :ghpull:`684` for implementation changes.
+
+Configurable kernel managers when executing notebooks
++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+The kernel manager can now be optionally passed into the ``ExecutePreprocessor.preprocess`` and the ``executenb`` functions as the kwarg ``km``. This means that the kernel can be configured as desired before beginning preprocessing.
+
+This is useful for executing in a context where the kernel has external dependencies that need to be set to non-default values. An example of this might be a Spark kernel where you wish to configure the spark cluster location ahead of time without building a new kernel.
+
+See :ghpull:`852` for implementation changes.
+
+Surfacing exporters in front-ends
++++++++++++++++++++++++++++++++++
+
+Custom exporters are now exposed for front-ends, including classic notebook, to consume. As an example, this means that latex exporter will be made available for latex 'text/latex' mimetype for the Download As interface.
+
+See :ghpull:`759` and :ghpull:`864` for implementation changes.
+
+Raw Templates
++++++++++++++
+
+Template exporters can now be assigned raw templates as string attributes by setting the raw_template variable.
+
+.. code-block::
+
+  class AttrExporter(TemplateExporter):
+      # If the class has a special template and you want it defined within the class
+      raw_template = """{%- extends 'rst.tpl' -%}
+  {%- block in_prompt -%}
+  raw template
+  {%- endblock in_prompt -%}
+      """
+  exporter_attr = AttrExporter()
+  output_attr, _ = exporter_attr.from_notebook_node(nb)
+  assert "raw template" in output_attr
+
+See :ghpull:`675` for implementation changes.
+
+New command line flags
+++++++++++++++++++++++
+
+The ``--no-input`` flag will apply metadata changes to input cells to mark them as hidden. This is great for notebooks which generate "reports" where you want the code that was executed to not appear by default.
```

Comment:
`--no-input` doesn't affect notebook metadata, in fact it has no use case if the output is a notebook. It works by filtering out the input blocks inside the templates. I'll make a change to reflect the mechanism.